src/DurableTask.Generators/AzureFunctions/SyntaxNodeUtility.cs

```diff
@@ -144,6 +152,27 @@ public static bool TryGetQualifiedTypeName(SemanticModel model, MethodDeclaratio
             return true;
         }
 
+        public static string GetRenderedTypeExpression(ITypeSymbol? symbol, bool supportsNullable)
+        {
+            if (symbol == null)
+            {
+                return supportsNullable ? "object?" : "object";
+            }
+
+            if (supportsNullable && symbol.IsReferenceType && symbol.NullableAnnotation != NullableAnnotation.Annotated)
+            {
+                symbol = symbol.WithNullableAnnotation(NullableAnnotation.Annotated);
+            }
+
+            string expression = symbol.ToString();
+            if (expression.StartsWith("System.") && symbol.ContainingNamespace.Name == "System")
+            {
+                expression = expression.Substring("System.".Length);
```

Comment:
We do this because the `System` namespace is implicitly imported by the compiler?